CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [0.4.3] - 2022-03-15
+
+### Features
+
+- Add the output of the number of rows imported after successfully importing data (#1401 @Manny-op)
+- Code Refactoring (#1366 @Cupid0320; #1378 @wuteek; #1418 @prashantpaidi; #1420 @shiyoubun)
+
+### Bug Fixes
+- Writing null to a non-null column fails. (#1341 @vagetablechicken)
```

Comment:
is this the correct one? seems the PR title is very different

Reply:
> is this the correct one? seems the PR title is very different

The #1340 is merged to main, #1341 is merged to branch-0.4. relevant issue is #1271